Cargo.toml

```diff
@@ -19,6 +19,7 @@ rust-version = "1.70"
 [workspace.dependencies]
 anyhow = "1.0.68"
 assert_matches = "1.5.0"
+async-rx = { git = "https://codeberg.org/jplatte/async-rx", rev = "4ba058835e947a316d859f3b46e0b86c41754df7" }
```

Comment:
Could we use a release instead? Friends don't let friends use git dependencies (or at least, spread them :p).

Reply:
I'm planning to do one today or tomorrow. I think that was Ivan's plan to have a release before merging anyways.

Reply:
Exactly, I wanted to test it in real-life before doing an `async-rx` release.

Reply:
`async-rx` 0.1.1 has been released. Patch updated. Resolving this conversation.